swagger_server/util.py

```diff
@@ -4,6 +4,8 @@
 import typing
 from swagger_server import type_util
 
+def get_swagger_class_enum_values(klass):
+    return [getattr(klass,i) for i in dir(klass) if not i.startswith("_") and isinstance(getattr(klass,i), str)]
```

Comment:
This file was generated by swagger.  That said, I don't expect we'll re-generate the entire server from swagger, and our gradle target doesn't re-generate it.  I think it's okay then, right?